L/Lua/build_tarballs.jl

```diff
@@ -0,0 +1,40 @@
+using BinaryBuilder
+
+name = "Lua"
+version = v"5.3.5"
+
+sources = [
+    "https://www.lua.org/ftp/lua-5.3.5.tar.gz" =>
+        "0c2eed3f960446e1a3e4b9a1ca2f3ff893b6ce41942cf54d5dd59ab4b3b058ac",
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/lua-*
+
+if [[ ${target} == *-apple-* ]]; then
+    MAKE_TARGET=macosx
+elif [[ ${target} == *-freebsd* ]]; then
+    # NOTE: The FreeBSD ports build also uses the bsd instead of the freebsd target,
+    # since it hard-codes fewer things, apparently
+    MAKE_TARGET=bsd
+elif [[ ${target} == *-mingw* ]]; then
+    MAKE_TARGET=mingw
+else
+    MAKE_TARGET=linux
+fi
+
+make -j${nproc} ${MAKE_TARGET}
+make install INSTALL_TOP=${WORKSPACE}/destdir
```

Comment:
```suggestion
make install INSTALL_TOP=${prefix}
```
but see also https://github.com/JuliaPackaging/Yggdrasil/pull/226#issuecomment-557836659